.github/workflows/pull-request-checks.yaml

```diff
@@ -125,10 +127,13 @@ jobs:
           submodules: recursive
 
       - name: Install bison and flex
-        run: "%SCRIPT_DIR%\\install-bison.bat"
+        run: "${{env.SCRIPT_DIR}}\\install-bison.bat"
```

Comment:
Why this change? Should be explained.

Reply:
I played around a little bit, and it appears this is the correct syntax.

`env`, in the context of github actions, is setting a *workflow environment variable*, not a shell one. If this is changed to a powershell syntax variable reference, we get all sorts of syntax errors, and this stack overflow answer seems to be supporting the previous statement: https://stackoverflow.com/questions/63467662/github-action-define-workflow-level-environment-variable-between-jobs